src/main/java/ch/njol/skript/expressions/ExprArgument.java

```diff
@@ -82,6 +82,7 @@ public class ExprArgument extends SimpleExpression<Object> {
 	private Argument<?> argument;
 
 	private int ordinal = -1; // Available in ORDINAL and sometimes CLASSINFO
+	private boolean SUPPORTS_UNKNOWN_EVENT;
 	
```

Comment:
Make this a `private static boolean BLAH = Skript.classExists` atm you're redefining this boolean each time the method gets called when this should be done once when the class is loaded.